app.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# project template
+# Copyright(C) 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Graph backup logic for the Thoth project."""
+
+import logging
+import os
+import subprocess
+from datetime import datetime
+from thoth.storages import __version__ as __storages__version__
+from thoth.storages import DatabaseBackupStore
+
+_LOGGER = logging.getLogger("thoth.graph_backup_job")
+__version__ = f"0.6.0+thoth_storage.{__storages__version__}"
+KNOWLEDGE_GRAPH_HOST = os.getenv("KNOWLEDGE_GRAPH_HOST", "localhost")
+KNOWLEDGE_GRAPH_PORT = os.getenv("KNOWLEDGE_GRAPH_PORT", "5432")
+KNOWLEDGE_GRAPH_USER = os.getenv("KNOWLEDGE_GRAPH_USER", "postgres")
+KNOWLEDGE_GRAPH_DATABASE = os.getenv("KNOWLEDGE_GRAPH_DATABASE", "postgres")
+os.environ["PGPASSWORD"] = os.getenv("KNOWLEDGE_GRAPH_PASSWORD", "postgres")
+
+
+def store_backup_to_ceph(backup_file: str) -> str:
+    """Upload the backup file to ceph."""
+    adapter = DatabaseBackupStore()
+    adapter.connect()
+    document_id = adapter.store_backup(os.path.abspath(backup_file), backup_file)
+    return document_id
+
+
+def main():
+    """Perform graph backup job."""
+    _LOGGER.debug("Debug mode is on.")
+
+    backup_file_name = f"pg_dump-{datetime.now().strftime('%s')}"
+    _LOGGER.info("The database backup would be taken into: %r.", backup_file_name)
+
+    _LOGGER.debug("Starting the subprocess to take the dump from database.")
```

Comment:
Maybe `info` so we see what's going on in the logs (backup will take some time).